third_party/build_deps.sh

```diff
@@ -0,0 +1,271 @@
+#!/bin/bash -xe
+
+ROOT_DIR=$(pwd)
+export INSTALL_PREFIX=${INSTALL_PREFIX:-/usr/local}
```

Comment:
I have general comments about this file:

1. Shouldn't this belong to DALI_deps?
2. You already have -e in the options. You don't need to chain every line with && since the script will be aborted on first error already
3. I'd probably create a single script for every dependency and this script will invoke each of them (probably all of that could live in DALI_deps)

I am thinking about the scenario that a dependency update changes the way to install it. I believe it'd make more sense if the installation script is also part of the DALI_DEPS version

Reply:
Bit of agree, we already moved patches there, which in fact are code written by us, on the other hand that repo is supposed to hold only references to deps.

Reply:
> Shouldn't this belong to DALI_deps?

I need to sort out the licensing internally first.

> You already have -e in the options. You don't need to chain every line with && since the script will be aborted on first error already

Done

> I'd probably create a single script for every dependency and this script will invoke each of them (probably all of that could live in DALI_deps)

Won't it introduce too much fragmentation? I would check if we can move the main script to the deps repo first.